source/routing/tests/raptor_test.cpp

```diff
@@ -3238,3 +3238,37 @@ BOOST_AUTO_TEST_CASE(reverse_pass_midnight_with_boardings){
     result = compute(5);
     BOOST_REQUIRE_EQUAL(result.size(), 0);
 }
+
+BOOST_AUTO_TEST_CASE(stay_in_pass_midnight){
```

Comment:
can you add a global test comment (even if it's quite clear with the name and dataset)